stdlib/Test/test/runtests.jl

```diff
@@ -1753,3 +1753,18 @@ module M54082 end
         @test only(result) isa Test.Fail
     end
 end
+
+# Note: if the stream of numbers generated by the given RNG changes, then we'll need to
+# change the reference results as well and that's fine.
```

Comment:
Do these tests really need to test the numbers returned by the rng? I think it's sufficient to test that `rand() == rand(copy_of_rng_passed_to_testset)`, which would make these tests stable in face of changes of the RNG stream.

Reply:
Done.